docs/DeviceCreationGuide.md

```diff
@@ -0,0 +1,182 @@
+
+  
+## Step-by-Step Guide for Device Creation in Specter
+
+### Add a new device
+Select signing device
+![image](https://user-images.githubusercontent.com/47259243/223428531-2f3a04d4-177d-4626-8108-b66234892541.png)
+Upload public keys
+![image](https://user-images.githubusercontent.com/47259243/223427859-c06faec5-78ab-4592-9ba6-4018978280cc.png)
+
+### Select how to connect to Bitcoin network
+![image](https://user-images.githubusercontent.com/47259243/223425374-a3e68ac7-2bdb-48fe-a53b-59f235c59bd1.png)
+Electrum server or...
+![image](https://user-images.githubusercontent.com/47259243/223426046-dd225f00-ba18-45cb-871a-40efd7eefc1e.png)
+...via Bitcoin Core node.
+![image](https://user-images.githubusercontent.com/47259243/223426366-c3ba758a-34c4-4ce1-8aae-cf0cc335a892.png)
+
+
+## Understanding Derivation Paths
+
+### Concept Explanation
+
+Understanding derivation paths is fundamental to managing the security of your digital assets. In this section, we'll provide you with an overview of what derivation paths are and why they matter. We'll also introduce key paths like BIP 44 (for multi-account hierarchy), BIP 49 (for SegWit compatibility), and BIP 84 (for native SegWit addresses). Each of these paths caters to different Bitcoin address types and plays a crucial role in organizing and securing your cryptocurrencies, especially within hardware wallets.
+
+By default, Specter Wallets are set up with:
+
+  - BIP 44 for traditional multisig wallets.
+  - BIP 49 or BIP 84 for SegWit singlesig wallets.
+
+These default settings cover the needs of most users, simplifying the wallet setup and usage process. However, understanding these paths can enhance your ability to tailor the wallet to your specific needs, especially if you have advanced security considerations.
+
+### Challenges and Best Practices
+
+1. **Complexity:** 
+   - Derivation paths, especially when considering various Bitcoin address types like BIP 44, BIP 49, and BIP 84, can be intricate. The challenge lies in comprehending the nuances of each path and selecting the one that aligns with your specific use case. Best practice here is to educate yourself thoroughly and seek expert advice if needed.
+
+2. **Compatibility:**
+      - Using the wrong derivation path can lead to compatibility issues, making it challenging to access your funds. It's crucial to ensure that the path you choose is supported by your wallet software and the services you intend to use. Staying informed about updates and changes in the cryptocurrency ecosystem is essential to avoid compatibility pitfalls.
+      - To assist with this, common derivation paths for different wallets can be found at [Wallets Recovery](https://walletsrecovery.org/). This resource can be useful for understanding the standard practices of various wallets and ensuring compatibility.
+      - Ensure that the path you choose is supported by your wallet software and the services you intend to use.
```

Comment:
Inconsistent bullet-points: Please settle for one approach. I like the one in 1.
So maybe just remover some tabs here and make. sure it works in the preview.